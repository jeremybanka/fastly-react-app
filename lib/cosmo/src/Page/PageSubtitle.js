/* @flow */

import * as React from "react";
import { Box } from "../";

type Props = { children: React.Node };

const PageSubtitle = (props: Props) => (
  <Box color="grays.5" fontSize="sm" marginTop="sm">
    {props.children}
  </Box>
);

export default PageSubtitle;
