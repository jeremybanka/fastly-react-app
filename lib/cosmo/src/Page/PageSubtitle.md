The `<Page.Subtitle />` component should be nested inside of a `<Page />`.
