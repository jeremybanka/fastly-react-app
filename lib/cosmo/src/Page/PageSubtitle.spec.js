/* @flow */

import React from "react";
import { shallow, render } from "enzyme";

import PageSubtitle from "./PageSubtitle";

describe("PageSubtitle test suite", () => {
  it("renders with className .PageSubtitle", () => {
    expect(
      shallow(<PageSubtitle>My Subtitle</PageSubtitle>).is(".PageSubtitle")
    ).toBe(true);
  });

  it("renders the stored snapshot", () => {
    expect(render(<PageSubtitle>My Subtitle</PageSubtitle>)).toMatchSnapshot();
  });
});
