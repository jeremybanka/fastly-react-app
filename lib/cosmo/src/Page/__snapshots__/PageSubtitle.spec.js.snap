// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSubtitle test suite renders the stored snapshot 1`] = `
<div
  class="PageSubtitle"
>
  My Subtitle
</div>
`;
